wagtailinventory/forms.py

```diff
@@ -1,43 +1,43 @@
-from __future__ import absolute_import, unicode_literals
-
 from django import forms
 from django.db.models.fields import BLANK_CHOICE_DASH
 from django.forms import formset_factory
 
+from wagtailinventory.models import PageBlock
+
+
 try:
     from wagtail.core.models import Page
 except ImportError:  # pragma: no cover; fallback for Wagtail <2.0
     from wagtail.wagtailcore.models import Page
```

Comment:
Looks like this was missed.

```suggestion
from wagtail.core.models import Page
```

Reply:
Removed old Wagtail 1 import